src/test_l1_tvSettings.c

```diff
@@ -9968,9 +9969,9 @@ void test_l1_tvSettings_negative_GetColorTemp_Rgain_onSource (void)
     for (unsigned int i = 0; i < tvColorTemp_MAX; i++)
     {
         platformFlag = false;
-        for (unsigned int j = 0; j < colorTempCount ; j++ )
+        for (unsigned int j = 0; j < colorTempCount; j++ )
         {
-            snprintf(keyValue, UT_KVP_MAX_ELEMENT_SIZE, "tvSettings/ColorTemperature/index/%d", i);
+            snprintf(keyValue, UT_KVP_MAX_ELEMENT_SIZE, "tvSettings/ColorTemperature/index/%d", j);
             colorTempValue = (tvColorTemp_t) UT_KVP_PROFILE_GET_UINT32(keyValue);
             if(colorTempValue == (tvColorTemp_t)i)
             {
```

Comment:
"In line number 9984, are we supposed to pass the index i instead of colorTempValue? This is because we intend to pass the unsupported value."

Reply:
updated